internal/cmd/internal/olmv1/catalog_create.go

```diff
@@ -13,7 +14,10 @@ import (
 
 // NewCatalogCreateCmd allows creating a new catalog
 func NewCatalogCreateCmd(cfg *action.Configuration) *cobra.Command {
-	i := v1action.NewCatalogCreate(cfg.Client)
+	fmt.Println("lmao", cfg)
+	cfg.Load()
```

Comment:
The question is why https://github.com/operator-framework/kubectl-operator/blob/main/internal/cmd/root.go#L61 does not work for us?

Reply:
I think It was working at some point and with some recent changes it stopped working.

Reply:
I thin we are doing something not right in https://github.com/operator-framework/kubectl-operator/blob/main/internal/cmd/root.go#L61

Reply:
It should work now